services/admin.py

```diff
@@ -15,27 +16,81 @@ class ServiceClientIdInline(admin.StackedInline):
     verbose_name_plural = "client ids"
 
 
+class AllowedDataFieldsFilter(admin.SimpleListFilter):
+    title = _("allowed data fields")
+    parameter_name = "empty"
+
+    def lookups(self, request, model_admin):
+        names = (
+            AllowedDataField.objects.all()
+            .order_by("field_name")
+            .values_list("field_name", flat=True)
+        )
+        return [
+            ("empty", _("Empty")),
+        ] + [(name, name) for name in names]
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value == "empty":
+            return queryset.filter(allowed_data_fields__isnull=True)
+        elif value:
+            return queryset.filter(allowed_data_fields__field_name=self.value())
+        return queryset
+
+
 @admin.register(Service)
 class ServiceAdmin(TranslatableAdmin, GuardedModelAdmin):
-    list_display = ("indicate_profile_service",)
+    list_display = (
+        "indicate_profile_service",
+        "_allowed_data_fields",
+        "_client_ids",
+        "idp",
+        "_gdpr",
+    )
+    list_filter = (AllowedDataFieldsFilter, "idp")
+    search_fields = (
+        "name",
+        "translations__title",
+        "client_ids__client_id",
+    )
+    ordering = ("name",)
+
     inlines = [ServiceClientIdInline]
 
     def indicate_profile_service(self, obj):
-        result = str(obj)
         if obj.is_profile_service:
-            result = format_html(
-                '<span title="{}">⚙️ {}</span>', _("Profile service"), result
+            return format_html(
+                '<span title="{}">⚙️ {}</span>', _("Profile service"), obj.name
             )
-        return result
+        return obj.name
 
     indicate_profile_service.short_description = str(Service._meta.verbose_name)
```

Comment:
This can be replaced with:

```python
@admin.display(description=str(Service._meta.verbose_name))
def indicate_profile_service(self, obj):
   ...
```